src/getHumanAge.test.js

```diff
@@ -7,4 +7,36 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return [0, 0] if cat or dog age <=0', () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+    expect(getHumanAge(-15, -24)).toEqual([0, 0]);
+    expect(getHumanAge(-15, 15)).toEqual([0, 1]);
+    expect(getHumanAge(15, -24)).toEqual([1, 0]);
+  });
+
+  test('should returns 0 if cat or dog age <= 14', () => {
+    expect(getHumanAge(14, 14)).toEqual([0, 0]);
+  });
+
+  test('correctly converts cat and dog age to 1 human year', () => {
```

Comment:
'correctly' bad word, non-informative, what about 'should return 1 for cat and dog when they are 23 years old'?
so, here u should add one expect for [23, 23]
check here and in another tests